inference-engine/src/offline_transformations/include/generate_mapping_file.hpp

```diff
@@ -0,0 +1,32 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <memory>
+
+#include <ngraph/pass/graph_rewrite.hpp>
+
+namespace ngraph {
+namespace pass {
+
+class GenerateMappingFile;
+
+}  // namespace pass
+}  // namespace ngraph
+
+/**
+ * @brief Generate mapping file based on output tensor names.
+ */
+
+class ngraph::pass::GenerateMappingFile: public ngraph::pass::FunctionPass {
```

Comment:
Do we have a plan to use mapping file in future IR versions?

Reply:
We will discuss it on our upcoming IR V11 meeting. But the overall trend is to remove this file in 22.1.